completions/bash/crio

```diff
@@ -29,6 +29,7 @@ h
 --cni-config-dir
 --cni-default-network
 --cni-plugin-dir
+--collection-period
```

Comment:
Perhaps? Not entirely sure, though.

```suggestion
--metrics-collection-period
```

Reply:
I guess `collection-period` is much more suitable since it refers to both stats and metrics collection. see https://github.com/cri-o/cri-o/pull/7796#discussion_r1567830047